woocommerce/woocommerce
# Add z-index to onsale for twentyseventeen

### All Submissions:






### Changes proposed in this Pull Request:
The onsale label is missing a z-index for the twentyseventeen theme, twentynineteen has one, this PR adds a z-index so the on sale label is not hidden behind the main image.



### How to test the changes in this Pull Request:

1. Enable Twenty Seventeen theme
2. Put a product on sale
3. Open the product single page
4. Ensure the onsale badge is showing over the product image.

### Other information:




### Changelog entry

> Fix - Twenty Seventeen On Sale badge hidden behind product image.
